_posts/databases/postgresql/timescaledb/2000-01-01-downsampling.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Downsampling
+nav: Downsampling
+modified_at: 2022-01-21 00:00:00
+tags: timescale databases postgresql extensions downsampling
+---
+
+Downsampling data allows to reduce the amount of data by simplifying the old values.
+
+## Example
+
+Bellow an example of a logic of downsampling. It is based on TimescaleDB documentation.
```

Comment:
If it's based on the official doc, please make `TimescaleDB documentation` a link to the said page

```suggestion
Below an example of a logic of downsampling. It is based on TimescaleDB documentation.
```